rend3/src/setup.rs

```diff
@@ -346,10 +346,14 @@ pub struct InstanceAdapterDevice {
 }
 
 /// Creates an Instance/Adapter/Device/Queue using the given choices. Tries to get the best combination.
+///
+/// **NOTE:** Some adapters will not advertise all of its supported features. The `additional_features`
+/// parameter can be used to explicitly request additional features during device creation.
 pub async fn create_iad(
     desired_backend: Option<Backend>,
     desired_device: Option<String>,
     desired_mode: Option<RendererMode>,
+    additional_features: &[Features],
```

Comment:
Features is a bitmask, so we don't need an array here, just a single `Features`

Reply:
Oops, you're right! Should be fixed now.